Makefile.am

```diff
@@ -23,3 +23,8 @@ ACLOCAL_AMFLAGS = -Im4/autotools
 SUBDIRS = m4 include lib src doc man
 dist_doc_DATA = README.md
 dist_noinst_DATA = tlog.spec
+
+# Remove coverage data files and clang results generated by CI runs
+clean-local:
+	find '$(top_builddir)' -name '*.gcno' -delete -or -name '*.gcda' -delete
+	find '$(top_builddir)' -name 'clang_output_*' -delete
```

Comment:
Ah, wait, we don't need to look for 'clang_output_*' files. They're put into the source root directory. A simple "rm" will do. Please amend. Otherwise this looks fine.